pymc/distributions/timeseries.py

```diff
@@ -12,11 +12,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
+from typing import Optional, Union
+
 import aesara.tensor as at
 import numpy as np
 
 from aesara import scan
-from scipy import stats
+from aesara.tensor.random.basic import RandomVariable
+
+import pymc as pm
```

Comment:
Replace this with specific imports that you need.

Reply:
Will do. For reference I have this same confusion for the entire library. We use `import pymc as pm` in many other submodules

https://github.com/pymc-devs/pymc/blob/main/pymc/distributions/multivariate.py#L42-L53
https://smartinference.slack.com/archives/C6AKTNSBS/p1642434564015100

Reply:
done